extensions/agroal/runtime/src/main/java/io/quarkus/agroal/runtime/DataSourceJdbcRuntimeConfig.java

```diff
@@ -115,6 +115,12 @@ public interface DataSourceJdbcRuntimeConfig {
      */
     Optional<String> validationQuerySql();
 
+    /**
+     * Enable connection validation on borrow.
```

Comment:
```suggestion
     * Allows connection validation prior to acquisition (foreground validation) overriding any idle timeout.
     * <p>
     * Because of the overhead of performing validation on every call, it’s recommended to rely on default idle validation instead, and to leave this to `false`.
```

Reply:
I will incorporate your suggestion. Thx for the review.